installer/bundle/bundle_skel_Linux.sh

```diff
@@ -620,6 +621,10 @@ case "$installMode" in
             if [ $? -eq 0 ]; then
                 shouldInstall_omi
                 pkg_upd_list $OMI_PKG omi $?
+            else
+                pkg_add_list $OMI_PKG omi
+                pkg_add ${ADD_PKG_QUEUE}
+                OMI_EXIT_STATUS=$?
```

Comment:
I'm not sure you need "check_if_package_is_installed". At least in the OMS version, that's handled as part of "shouldInstall_omi". If it's NOT installed, then "shouldInstall_omi" says "install it".

But that depends on exactly how that code is written ...